### PR TITLE
Add initial search documentation for solr + fulltext search

### PR DESCRIPTION
**Pull request by HusseinKabbout**

### CODE REVIEW

File: src/topics/Search.md [hunk 1]
```diff
@@ -118,4 +118,150 @@ Note: The `qwc2-demo-app` (also used by the `qwc-map-viewer-demo` docker image)
...
+                'default' AS tenant,
+                (array_to_json(array_append(ARRAY[id_name::text], id_type::text)))::text AS idfield_meta,
+                (st_asgeojson(st_envelope(geom), 0, 1)::json -> 'bbox')::text AS bbox
+            FROM index_base">
+        </entity>
+    </document>
+</dataConfig>
+```
+
+The next table shows how the values, that are surrounded by {}, need to be defined.
```
Comment (manisandro) [inline src/topics/Search.md:167]: `{}`

File: src/topics/Search.md [hunk 2]
```diff
@@ -118,4 +118,150 @@ Note: The `qwc2-demo-app` (also used by the `qwc-map-viewer-demo` docker image)
...
+    "name": "SEARCH_NAME",
+    "filter_word": "OPTIONAL_SEARCH_FILTER",
+    "table_name": "SCHEMA.SEARCH_TABLE_NAME",
+    "geometry_column": "GEOMETRY_FIELD",
+    "search_id_col": "PRIMARY_KEY"
+}
+```
+
+The `filter_word` field can be specified to activate / deactivate searches,
+if you have configure multiple searches for one map.
```
Comment (manisandro) [inline src/topics/Search.md:227]: map -> theme

File: src/topics/Search.md [hunk 3]
```diff
@@ -118,4 +118,150 @@ Note: The `qwc2-demo-app` (also used by the `qwc-map-viewer-demo` docker image)
...
+        "provider": "solr",
+        "default": [],
+        "layers": {
+            "QGIS_LAYER": "SEARCH_NAME"
+        }
+    }
+]
+```
+
+When activating a search to a map, you can either:
```
Comment (manisandro) [inline src/topics/Search.md:252]: map -> theme

**Comment by tpo** [comment]
nice! +1 @HusseinKabbout !

**Pull request merged**